client/index.js

```diff
@@ -31,12 +31,18 @@ addFilter(
 			path: '/payments/connect',
 			wpOpenMenu: menuID,
 			breadcrumbs: [ rootLink, __( 'Connect', 'woocommerce-payments' ) ],
+			navArgs: {
```

Comment:
This will replace the `WooCommerce Payments` menu with `Connect` under the extensions menu.

<img width="272" alt="Screen Shot 2020-12-01 at 11 27 08 AM" src="https://user-images.githubusercontent.com/10561050/100769268-cca8f500-33c9-11eb-95ae-c5bab14212a8.png">

Reply:
Thanks @joshuatf, fixed in https://github.com/Automattic/woocommerce-payments/pull/1136/commits/39d45d6acaba165ddcad6a4e4fc08530a3b361e1

Reply:
And added a better check in https://github.com/Automattic/woocommerce-payments/pull/1136/commits/26af034037debe38c3fdf47b78a657d037297d13